internals/contrib/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -ex
+
+FEATURES="std alloc"
+
+if [ "$DO_LINT" = true ]
+then
+    cargo clippy --frozen --all-features --all-targets -- -D warnings
```

Comment:
Why use `--frozen` over `--locked`, is it because we know the order of `CRATES` so we know that cargo has already been run with `--locked` so doesn't need to hit the network again?

Reply:
I previously thought `--frozen` does something else. I changed it to `--locked` but forgot in this case.